doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -189,7 +290,11 @@ CREATE SOURCE mz_source
 Materialize performs an initial sync of all tables in the publication before it starts ingesting change events.
 {{</ note >}}
 
-## Create a materialized view
+## Step 5. Monitor the initial snapshot
+
+## Step 6. Explore data in Materialize
+
+## Next steps
 
 Any materialized view that depends on replication subsources will be incrementally updated as change events stream in, as a result of `INSERT`, `UPDATE` and `DELETE` operations in the original Postgres database.
```

Comment:
"replication subsources" is a new term here--this likely needs to be tightened up a bit/clarified.